templates/repo/issue/view_content/sidebar.tmpl

```diff
@@ -354,11 +354,12 @@
 								</div>
 							{{end}}
 							<button class="ui fluid button poping up issue-start-time" data-content='{{.i18n.Tr "repo.issues.start_tracking"}}' data-position="top center" data-variation="small inverted">{{.i18n.Tr "repo.issues.start_tracking_short"}}</button>
-							<div class="ui mini modal issue-start-time-modal">
+							<div class="ui tiny modal issue-start-time-modal">
 								<div class="header">{{.i18n.Tr "repo.issues.add_time"}}</div>
 								<div class="content">
 									<form method="POST" id="add_time_manual_form" action="{{$.RepoLink}}/issues/{{.Issue.Index}}/times/add" class="ui action input fluid">
 										{{$.CsrfTokenHtml}}
+										<input type="date" name="created">
```

Comment:
I think it would be a good idea to add `pattern="\d{4}-\d{2}-\d{2}"` according to https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/date#handling_browser_support. Otherwise, a correct date could be added in an incorrect format that would then subsequently be reset to today.

Additionally, we could even think about using a maximum value of "today" or "tomorrow", as future times should in my opinion not be possible to be added. (Maybe except for timezones, hence why I would argue to leave "tomorrow" as a valid option). The problem with that is that the HTML standard does not support "today" or "tomorrow" as values and hence these values would need to be inserted on the server-side.
Thoughts on the maximum value?

Reply:
Regrading your proposal to add pattern="...". I am not sure that is necessary to do. I look through Yandex Browser, Firefox, Google Chrome, Opera, Safary. Default behaviour of browsers while adding input type=date looks reasonable. I can't add incorrect date and can't add extra range values.

Yandex Browser
<img width="544" alt="Screenshot 2021-08-30 at 17 11 44" src="https://user-images.githubusercontent.com/4449080/131353459-99255457-2773-4c06-9d06-c0efa01f3c87.png">

Firefox
<img width="545" alt="Screenshot 2021-08-30 at 17 11 59" src="https://user-images.githubusercontent.com/4449080/131353498-b0d803d6-cdfa-40f1-9014-d925c9ea9d68.png">

Google Chrome
<img width="410" alt="Screenshot 2021-08-30 at 17 12 37" src="https://user-images.githubusercontent.com/4449080/131353532-635d23c9-2711-487d-a86c-be8ab8ebed3a.png">

Opera
<img width="546" alt="Screenshot 2021-08-30 at 17 13 18" src="https://user-images.githubusercontent.com/4449080/131353586-1bcde301-ae59-430c-9dc3-6979d778047a.png">

Safari
<img width="544" alt="Screenshot 2021-08-30 at 17 13 51" src="https://user-images.githubusercontent.com/4449080/131353597-ab31cede-2b2a-415d-acea-f5b79b4c20a1.png">

Second part of your review to use maximum value "today" or "tomorrow" looks good. I try to find how to implement it  and add it to this PR.